src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -219,6 +228,15 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             }
         }
 
+        if (isSetAsideApplicationRequest(sscsCaseData)
+                && caseDetails.getState().equals(State.DORMANT_APPEAL_STATE)) {
```

Comment:
Minor - for consistency
```suggestion
                && State.DORMANT_APPEAL_STATE.equals(caseDetails.getState())) {
```

Reply:
Also could do with a test covering when this is not State.DORMANT_APPEAL_STATE

https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSTCA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs%2Fccd%2Fpresubmit%2Ffurtherevidence%2Factionfurtherevidence%2FActionFurtherEvidenceAboutToSubmitHandler.java&view=list&pullRequest=3124&id=SSCSTCA

Reply:
Yep, will do.